src/nostr-livestream/spec/nip-53.md

```
@@ -0,0 +1,267 @@
+NIP-53
+======
+
+Live Activities
+---------------
+
+`draft` `optional`
+
+This NIP introduces event kinds to advertise live spaces and the participation of pubkeys in them.
+
+## Live Streaming
```

markdownlint-cli2 (0.18.1)

Heading style
Expected: setext; Actual: atx

(MD003, heading-style)